src/main/java/net/coderbot/iris/gui/screen/ShaderPackScreen.java

```diff
@@ -179,6 +181,20 @@ public void filesDragged(List<Path> paths) {
 		this.addedPackDialogTimer = 100;
 	}
 
+	private static void copyShaderPack(Path pack, String name) throws IOException {
+		Path target = Iris.SHADERPACKS_DIRECTORY.resolve(name);
+
+		// Make use of FileUtils to copy a directory with ease.
+		// Needs to convert the NIO Path to an IO File in order to work.
+		if (Files.isDirectory(pack)) {
+			FileUtils.copyDirectory(pack.toFile(), target.toFile());
```

Comment:
Not all `Path`s can be converted to `File`s, this should be at the very least checked and noted in a comment.

Reply:
Alright, and I feel like there could be a better, cleaner solution to copying a directory and its contents, so if you know of one let me know

Reply:
I've found a few examples that make use of `Files.walk`, but obviously it wouldn't be a one liner unless we have our own util class. I'm not seeing any one liners in the standard library but maybe I'm not looking hard enough. I wouldn't be opposed to that though, since relying on toFile isn't great, especially when we can be passed arbitrary `Path`s from the OS.